jme3-lwjgl3/src/main/java/com/jme3/system/lwjgl/LwjglWindow.java

```diff
@@ -271,12 +324,91 @@ public void callback(long args) {
             glfwSwapInterval(0);
         }
 
+        final Object[] icons = settings.getIcons();
+
+        if (icons != null) {
+
+            final GLFWImage[] images = imagesToGLFWImages(icons);
+            final GLFWImage.Buffer iconSet = GLFWImage.malloc(images.length);
+
+            for (int i = images.length - 1; i >= 0; i--) {
+                final GLFWImage image = images[i];
+                iconSet.put(i, image);
+            }
+
+            glfwSetWindowIcon(window, iconSet);
+
+            iconSet.free();
```

Comment:
I have no time for researching this problem. :( I'm going to an army in several days for 2 months.